src/components/TodoItem.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styles from './TodoItem.module.css';
+
+class TodoItem extends React.Component {
+  state = {
+    editing: false,
+  }
+
+  handleEdit = () => {
+    this.setState({
+      editing: true,
+    });
+  }
```

Comment:
You have a function for handling edits on your to-do items but have not implemented it as required by the curriculum. Please add the Edit implementation to your to-do items to allow for edit and updating of your to-do items as required.